src/alfasim_score/converter/alfacase/convert_plugin_data.py

```diff
@@ -55,7 +57,33 @@ def get_fluid_id(self, fluid_name: str) -> int:
         """
         return self.get_all_annular_fluid_names().index(fluid_name)
 
-    def _build_annular_fluid_table(self, fluids_data: List[Dict[str, Any]]) -> AnnulusTable:
+    def _build_annular_temperature_table(self) -> AnnulusTemperatureTable:
+        """
+        Calculate the temperature of fluids based on temperature of formation coming from SCORE input file.
+        It uses the well vertical positions in order to interpolates the temperatures of formation and it maps
+        each measured depth to the temperature of formation in that position.
+        """
+        measured_depths = Array(
+            [
+                self.get_position_in_well(depth).GetValue()
+                for depth in self.score_input.read_well_trajectory()["md"]
+            ],
+            LENGTH_UNIT,
+        )
+        formation_temperature_data = self.score_input.read_formation_temperatures()
+        trajectory = self.score_input.read_well_trajectory()
+        interpolated_temperatures_y = np.interp(
```

Comment:
It seems that `formation_temperature_data["elevations"]` is quota and `trajectory["y"]` is TVD. Thus, it is using incompatible data. Is it correct?

Reply:
I suppose that's true! Fixed